bot/cogs/moderation/scheduler.py

```diff
@@ -411,3 +411,12 @@ def mod_log(self) -> ModLog:
         await time.wait_until(expiry)
 
         await self.deactivate_infraction(infraction)
+
+    @staticmethod
+    def _get_plain_user(user: MemberObject) -> str:
+        """Get plain string representation for `user` in embed."""
+        if isinstance(user, ProxyUser):
```

Comment:
Despite the parameter _is_ type-annotated, I feel catching `AttributeError`s or using `hasattr` is better than checking the type. Generally I prefer to not set restrictions on entire types unless I have to. All that is needed here is one attribute - we don't actually care what the type is as long as it can give us a name or nick. Maybe it doesn't matter since the type annotation already "restricts" it. What do you think?